generator/config/expression/pow.yaml

```diff
@@ -15,3 +15,16 @@ arguments:
         name: exponent
         type:
             - resolvesToNumber
+tests:
+    -
+        name: 'Example'
+        link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/pow/#example'
+        pipeline:
+            -
+                $project:
+                    variance:
+                        $pow:
+                            -
+                                # $stdDevPop: '$scores.score'
```

Comment:
I've checked the result is the same if the value is in an array or a single value

Reply:
Is this because StdDevPopOperator encodes as a single element, which utilizes its only `$expression` property (a list)? I suppose `$expressions` would be a more descriptive name for the property but I realize the class is auto-generated and derived from the name of the variadic arg itself.

In any event, I think it makes sense to capture this explanation in a YAML comment here to explain exactly why you're changing the expected pipeline.

IIRC, this kind of thing came up in previous PRs, since there are other operators that omit logic to encode using the "short" form.